dev/acf-export.json

```diff
@@ -800,6 +816,41 @@
                 "copy_to_clipboard": 0,
                 "display_type": "text"
             },
+            {
+                "key": "field_5df79878fc895",
+                "label": "Degree Outcome Data",
+                "name": "",
+                "type": "tab",
+                "instructions": "",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "placement": "top",
+                "endpoint": 0
+            },
+            {
+                "key": "field_5df79894fc896",
+                "label": "Display Degree Outcomes",
+                "name": "degree_display_outcomes",
+                "type": "true_false",
+                "instructions": "Determines if degree outcomes are displayed",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "message": "",
+                "default_value": 1,
```

Comment:
Talked to Roger about this on Friday and forgot to mention it, but he wants to default to not showing these imported data fields for now.